indico/modules/events/editing/templates/emails/submitter_confirmation_notification.txt

```diff
@@ -0,0 +1,7 @@
+{% extends 'events/editing/emails/base_email.txt' %}
+
+{% block subject -%}New submitter confirmation{%- endblock %}
+
+{% block body_text -%}
+Your revision changes have been {{ action }} by {{ submitter_name }}.
```

Comment:
What about two separate templates for the actions? I think the subject could be different based on the action as well (e.g. "Submitter confirmed changes" and "Submitter rejected changes"). It's a rather important difference, so I think having it in the subject is worth it.